test/addAction.test.js

```diff
@@ -0,0 +1,21 @@
+const { expect } = require('chai');
```

Comment:
Is this still needed?

Reply:
I'm unsure why we'd get rid of expect when we use that as an assertion in the test

Reply:
I was referring to the whole file. Do we still have an addClass???